pytest 1 failure - due to find_query changes from previous PR

def test_find_query():
>       assert bq_extract_data.find_query("test_bq_extract_data.py", "./tests") == "./tests/test_bq_extract_data.py"
E       AssertionError: assert None == './tests/test_bq_extract_data.py'
E        +  where None = <function find_query at 0x10d803620>('test_bq_extract_data.py', './tests')
E        +    where <function find_query at 0x10d803620> = bq_extract_data.find_query

--- a/src/data/tests/test_bq_extract_data.py
+++ b/src/data/tests/test_bq_extract_data.py
@@ -3,10 +3,13 @@
 import bq_extract_data
 
 def test_find_query():
-    assert bq_extract_data.find_query("test_bq_extract_data.py", "./tests") == "./tests/test_bq_extract_data.py"
+    # only returns .sql files, addresses issue 10 somewhat
+    assert bq_extract_data.find_query("test_bq_extract_data.py", "./tests") == None
     assert bq_extract_data.find_query("quer", "./tests") == "./tests/query.sql"
     # returns first file to match query_arg, bug or feature?
     assert bq_extract_data.find_query("", "./tests") == "./tests/test.sql"
+    # potential bug spotted
+    # assert bq_extract_data.find_query("query.sql", "./tests") == "./tests/query.sql"
 
 # test removing linebreaks from sql query file
 # add space for line breaks

